ktlint-cli/src/main/scripts/ktlint.bat

```diff
@@ -1,8 +1,10 @@
 @echo off
+setlocal
+
 REM By default this batch file and the "ktlint" are located in same directory. Note that "ktlint" is a JAR
 REM despite it is not being suffixed with extension ".jar". Please adjust path below when this batch file
 REM and the jar file are located in different directories.
-if not defined JAR_PATH set JAR_PATH=.\ktlint
+if not defined JAR_PATH set JAR_PATH=%~dp0ktlint
```

Comment:
Theoretically one would write `JAR_PATH="%~dp0ktlint"` to make sure that spaces are handled, but the quotes will become part of the `JAR_PATH` variable, and therefor can break future usages. The expansion is space-safe, and `JAR_PATH` can contain spaces. I tested it.